nixos/modules/programs/gnupg.nix

```diff
@@ -135,7 +135,7 @@ in
       # The SSH agent protocol doesn't have support for changing TTYs; however we
       # can simulate this with the `exec` feature of openssh (see ssh_config(5))
       # that hooks a command to the shell currently running the ssh program.
-      Match host * exec "${cfg.package}/bin/gpg-connect-agent --quiet updatestartuptty /bye > /dev/null"
+      Match host * exec "${cfg.package}/bin/gpg-connect-agent --quiet updatestartuptty /bye &>/dev/null"
```

Comment:
I would use `>/dev/null 2>&1`, which is portable: I don't know which shell will ssh use to run `exec` commands.